input/fsh/examples/CBTN_example.fsh

```diff
@@ -350,10 +350,10 @@ InstanceOf: NcpiFamilyRelationship
 Title: "An example family relationship based on data from CBTN"
 Usage: #example
 Description: "An example family relationship based on data from CBTN."
-* subject = Reference(PT-006SP660)
-* focus = Reference(PT-006SP675)
-* code = $family-role-code#MTH "mother"
-* status = #registered
```

Comment:
I didn't know what "registered" meant regarding an Observation resource for a family relationship. So I substituted it with "completed" - we should define the terms of the status since it is a required field.